ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -169,14 +169,18 @@ public struct Avatar: View {
         let shouldDisplayPresence = presence != .none
         let isRingVisible = state.isRingVisible
         let hasRingInnerGap = state.hasRingInnerGap
+        let ringThicknessToken: CGFloat = tokens.ringThickness
+        let ringInnerGapOffset = 0.5
         let isTransparent = state.isTransparent
         let isOutOfOffice = state.isOutOfOffice
         let initialsString: String = ((style == .overflow) ? state.primaryText ?? "" : Avatar.initialsText(fromPrimaryText: state.primaryText,
                                                                                                            secondaryText: state.secondaryText))
         let shouldUseCalculatedColors = !initialsString.isEmpty && style != .overflow
 
-        let ringInnerGap: CGFloat = isRingVisible && hasRingInnerGap ? tokens.ringInnerGap : 0
-        let ringThickness: CGFloat = isRingVisible ? tokens.ringThickness : 0
+        // Adding a 0.5 value offset to ringInnerGap & ringThickness to accommodate for a small space between
```

Comment:
update comment and also maybe we should move the constant closer to this block of code?